### PR TITLE
[docs] Text revisions configuring network policies

### PR DESCRIPTION
**Pull request by Lada7878**
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Text revisions configuring network policies.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/_data/deckhouse-ports.yml [hunk 1]
```diff
@@ -13,22 +13,22 @@ groups:
...
     description:
       en: Deckhouse controller webhook handler
-      ru: Webhook-обработчик контроллера Deckhouse
+      ru: Вебхук-обработчик контроллера Deckhouse
 
 - group: MastersToNodes
   description:
     en: Master to nodes traffic
-    ru: Трафик между мастер-узлами и остальными узлами
+    ru: Трафик от мастер-узлов к узлам
```
Comment (z9r5) [inline docs/documentation/_data/deckhouse-ports.yml:31]:
```suggestion
    ru: Трафик от мастер-узлов к узлам
```
@Lada7878 Поправь согласно политике пожалуйста

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Review by Zhbert — COMMENTED** [review]
Small fixes.

**Review by Zhbert — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [11118](https://github.com/deckhouse/deckhouse/pull/11118) to the branch [release-1.67](https://github.com/deckhouse/deckhouse/tree/release-1.67) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [11119](https://github.com/deckhouse/deckhouse/pull/11119) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Pull request merged**